deps/Makefile

```diff
@@ -263,7 +263,7 @@ endif
 else # NO_GIT
 $2_SRC_DIR = $1-$$($2_BRANCH)
```

Comment:
probably best to rename the file to use the SHA1 hash also, so we don't confuse the checksum system.